### ISSUE TITLE
Hide unfixable linting warnings 

### ISSUE THREAD

**Issue by mdefende** [uabrc/researcher-facing-MkDocs#27]
Related to pull request #25, some linting issues cannot be fixed. These are often related to the admonition syntax in conjunction with other surrounding entities that are not basic text as well as missing alt text in images. These linting warnings can be fixed on a per-line or per-file basis. Fixing on a whole-project basis I believe is possible, but I cannot find the documentation for it immediately.

To fix on a per-line basis:

```
<!-- markdownlint-disable <lint warning code> -->
deliberate space * in * emphasis
<!-- markdownlint-enable <lint warning code> -->
```

or 

```
<!-- markdownlint-disable-next-line -->
```

To fix on a per-file basis, add the following to the top of the file:

``` 
<!-- markdownlint-disable-file <lint warning code> -->
```

**Comment by wwarriner** [comment]
One drawback to this approach, as I understand, is the comments may show up in the generated HTML. Does that happen with mkdocs?

**Comment by mdefende** [comment]
At least in their linting docs, they say it does not get rendered as text on the HTML page.

**Comment by wwarriner** [comment]
Awesome! I'll take a stab at this later on
